src/v2/Components/__stories__/Molecules.story.tsx

```diff
@@ -0,0 +1,200 @@
+  },
+}
+
+const AppContainer = styled(Box)`
+  max-width: ${themeGet("breakpoints.lg")};
+`
+
+AppContainer.defaultProps = {
+  mx: "auto",
+  px: [2, 4],
+}
+
+const Page: React.FC = ({ children }) => {
+  return <AppContainer>{children}</AppContainer>
+}
+
+const MetadataExample = () => {
+  return (
+    <>
+      {/* TODO: Mixin a textTransform prop to text */}
+      <Text variant="xs" style={{ textTransform: "uppercase" }} mb={1}>
+        Category Title
+      </Text>
+
+      <Text variant="xl">Section Title or Headline</Text>
+
+      <Text variant="xl" color="black60" mb={1}>
+        Secondary Text
+      </Text>
+
+      <Text variant="lg" mb={1}>
+        Sub Headline
+      </Text>
+
+      <Text variant="sm" mb={4}>
+        Knausgaard vexillologist shabby chic, bitters vape vaporware live-edge
+        mixtape ennui raclette
+      </Text>
+
+      {/* TODO: Support n-column grids with a context provider as an alternative to sub-grid support? */}
+      <GridColumns>
+        <Column span={8}>
+          <Button width="100%">Button Title</Button>
+        </Column>
+      </GridColumns>
+    </>
+  )
+}
+
+const BioExample = () => {
+  return (
+    <>
+      <Text variant="xs" style={{ textTransform: "uppercase" }} mb={1}>
+        Bio
+      </Text>
+
+      <Text variant="sm" mb={2}>
+        With titles such as Susanne, No Bra (2006) and Anders pulling splinter
+        from his foot (2004), Wolfgang Tillmans’s oeuvre is distinguished by
+        unabashed emotion and a tension between strangeness and familiarity.
+        Using all the photographic technology at his disposal, Tillmans shoots
+        portraits, still-lifes, and landscapes in which the subjects range from
+        partially-nude friends in seemingly private moments to modest
+        arrangements of domestic items on windowsills. Tillmans’s abstractions
+        reveal studies in color, as seen in the magenta liquid lines of Urgency
+        XXIV (2006).
+      </Text>
+
+      <Text variant="xs" style={{ textTransform: "uppercase" }} mb={1}>
+        Stats
+      </Text>
+
+      <Text variant="sm">
+        £549k Auction Record
+        <br />
+        Blue Chip Representation
+      </Text>
+    </>
+  )
+}
+
+export const Page1 = () => {
+  return (
+    <Page>
+      <FullBleed>
+        <Box bg="black30" height={600} mb={4} />
+      </FullBleed>
+
+      <GridColumns>
+        <Column span={6}>
+          <Box size={80} border="1px solid" borderColor="black30" mb={1} />
+
+          <MetadataExample />
+        </Column>
+
+        <Column span={6}>
+          <BioExample />
+        </Column>
+      </GridColumns>
+
+      <Spacer my={12} />
+
+      <GridColumns>
+        <Column span={6}>
+          <MetadataExample />
+        </Column>
+
+        <Column span={6}>
+          <Box bg="black30" width="100%" height={480} />
+        </Column>
+      </GridColumns>
+
+      <ProgressDots variant="dash" amount={5} activeIndex={1} my={6} />
```

Comment:
Just for visual. Happy to make a rotating stepper component though when the need arises.